pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md

```diff
@@ -0,0 +1,444 @@
+
+![Green statuses on Database Control Panel](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql08.png){.thumbnail}
+
+### Authorise your IP address
+
+> [!warning]
+> For security reasons the default network configuration doesn't allow any incoming connections. 
+> To allow access from your application or development environment to your PostgreSQL database service, you must add your IP addresses to the service whitelist.
+
+#### Add your IPs to the DB authorised list
+
+Following the related documentation on how to [authorise the suitable IP addresses](https://docs.ovh.com/gb/en/publiccloud/databases/mongodb/managing-service/#configuring-authorised-ips), add your IPs to the authorised list.
+
+![Green statuses and authorized IPs setted on Database Control Panel](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql10.png){.thumbnail}
+
+### Test connection to the PostgreSQL DataBase
+
+An easy and quick way to test the connection is using the Postgre Command Line Interface (CLI) [psql](https://docs.postgresql.fr/14/app-psql.html){.external}.
+
+> 💡 Tip : If you are familiar with [Docker](https://www.docker.com/){.external} or [Kubernetes](https://kubernetes.io/){.external}, you can use the psql tool inside a postgres Docker [image](https://hub.docker.com/_/postgres){.external} instead of installing the tool locally.
+>
+> Examples :
+> ```sh
+> $ kubectl run postgres-client --image=postgres:latest -it --rm --restart=Never -- /bin/bash
+> ```
+> or
+> ```
+> $ docker run -it --rm postgres /bin/bash
+> ```
+>
+
+#### Connect to the database
+
+The useful parameters are:
+
+- The db Host, the db Port
+    - Get them from the "General Information tab"
+
+- The db Name
+    - Get it from the "Databases" tab, usually "defaultdb"
+
+- The db User
+    - Get it from the "Users" tab, usually "avnadmin"
+
+- The db Password
+    - Get it after you reset it.
+
+![Password reseted](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql11.png){.thumbnail}
+
+Now connect to the database with the following command
+
+```sh
+psql --dbname=defaultdb  --host=postgresql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net --port=20184 --username=avnadmin  --password 
+```
+Enter the password and press Enter
+
+<pre class="console"><code>$ psql --dbname=defaultdb  --host=postgresql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net --port=20184 --username=avnadmin  --password
+Password: 
+psql (14.1 (Debian 14.1-1.pgdg110+1))
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+Type "help" for help.
+
+defaultdb=> 
+</code></pre>
+
+Setup is done, your Managed PostgreSQL database is fully operational, let's go further and use it with <img src="images/strapi-logo-monogram-purple.png" alt="strapi logo" width="12px"> [Strapi](https://strapi.io/){.external}.
+
+### Before beginning, repare your environment
+
+What you need here is to check your nodejs and yarn versions.
```

Comment:
"What you need here is to check your `nodejs` and `yarn` versions."

Reply:
Done